msal4j-sdk/src/main/java/com/microsoft/aad/msal4j/OAuthAuthorizationGrant.java

```diff
@@ -15,74 +13,54 @@
 
 class OAuthAuthorizationGrant extends AbstractMsalAuthorizationGrant {
 
```

Comment:
q: do we have test coverage for extra query paramters?

Reply:
There are a few tests that use junk extra query parameters, but I don't think any test uses 'real' parameters that would be evaluated in an E2E. Is there an example of that sort of test in .NET?

Reply:
No, and we don't need an E2E test. We just need to ensure that the extra query parameters are appended to the URLs that MSAL hits, in particular the token endpoint.